tests/ut_expectations/ut_expectation_processor.stackOnUtFail.sql

```diff
@@ -0,0 +1,41 @@
+set termout off
```

Comment:
This test is failing on 11g:
https://travis-ci.org/utPLSQL/utPLSQL/jobs/249332892#L3583
And on 12.1:
https://travis-ci.org/utPLSQL/utPLSQL/jobs/249332893#L3565